docs/faq.md

```diff
@@ -382,17 +382,21 @@ Gradle: use `grade --debug -DjibSerialize=true` to enable more detailed logging
 
 ### How do I enable debugging?
 
-If using the `distroless/java` base image, then use the [`JAVA_TOOL_OPTIONS`](#how-do-i-set-parameters-for-my-image-at-runtime) to pass along debugging configuration arguments.  For example, to have the remote VM accept debug connections on port 5005, but not suspend:
+If using the `distroless/java` base image, then use the [`JAVA_TOOL_OPTIONS`](#how-do-i-set-parameters-for-my-image-at-runtime) to pass along debugging configuration arguments.  For example, to have the remote VM accept local debug connections on port 5005, but not suspend:
 ```
--agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=5005
+-agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=localhost:5005
 ```
 
-Then connect your debugger to port 5005 on the given host.  You can port-forward the container port to a localhost port for easy access.
+Then connect your debugger to port 5005 on your local host.  You can port-forward the container port to a localhost port for easy access.
 
 Using Docker: `docker run -p 5005:5005 <image>`
 
 Using Kubernetes: `kubectl port-forward <pod name> 5005:5005`
 
+Beware: in Java 8 and earlier, specifying only a port meant that
```

Comment:
@TadCordle mentioned avoid newlines? (I didn't get the reason though haha.)

Reply:
I mentioned avoiding explicit newlines so the auto-line wrapping can handle it. e.g. if you type this with manual newlines:

```
This is a paragraph. This is a paragraph. This is a paragraph. This is a 
paragraph. This is a paragraph. This is a paragraph. This is a paragraph.
This is a paragraph. This is a paragraph. This is a paragraph. This is a
paragraph. This is a paragraph. This is a paragraph. This is a paragraph.
This is a paragraph. This is a paragraph.
```

and resize the window of whatever you're using to view the formatted text, it could end up looking like:

```
This is a paragraph. This is a paragraph. This is a paragraph. This 
is a 
paragraph. This is a paragraph. This is a paragraph. This is a 
paragraph.
This is a paragraph. This is a paragraph. This is a paragraph. This 
is a
paragraph. This is a paragraph. This is a paragraph. This is a 
paragraph.
This is a paragraph. This is a paragraph.
```

I don't think this happens on GitHub specifically, so it's not a huge deal, but maybe in some IDEs it can look annoying.

Reply:
That said, it can also make code review more difficult for long paragraphs, since you may want to point out something in the middle of a paragraph, and breaking it up into lines lets you be more specific where you add github comments. So I'm not sure which you'd rather do (right now though I think most of Jib's documentation doesn't break paragraphs up into different lines).